utils.py

```diff
@@ -24,11 +27,110 @@ async def send_message(message, chat, footer_end=""):
     footer = footer.strip().replace(" ", "^ ^")
     footer = f"^{footer}^"
 
-    await chat.send_message(
-        f"{message}\n\n{footer}", creator=creator,
+    return await chat.send_message(
+        f"{message}\n\n{footer}",
+        creator=creator,
     )
 
 
+# Task reminder creator
+async def remind_task(ryver: Ryver, task: Task, minutes: int):
+    """
+    Apparently you can't either create reminders through pyryver or get the current session to send POST requests,
+    so this method creates another session and sets the task reminder before closing it.
+    """
+    console.log("Opening another session to create a reminder")
+
+    async with ClientSession(
+        auth=BasicAuth(getenv("RYVER_USER"), getenv("RYVER_PASS")),
+        raise_for_status=True,
+    ) as session:
+        url = ryver.get_api_url(
+            obj_type="tasks",
+            obj_id=task.get_id(),
+            action="UserNotification.Reminder.Create()",
+            format="json",
+        )
+        data = {"when": f"+{minutes} minutes"}
+        console.log("Creating reminder")
+        try:
+            async with session.post(url, json=data) as resp:
+                return (await resp.json())["d"]["id"]
+        except ContentTypeError:
+            pass
+        console.log("Reminder created")
+        await session.close()
+
+
+# Retrieves poll results and shows it on chat as a reply to the original poll message
+async def show_poll_results(chat: Chat, poll_id: str, bot_id: str):
+    console.log("Retrieving poll results")
+    try:
+        message = await retry_until_available(
+            chat.get_message,
+            poll_id,
+            timeout=5.0,
+            retry_delay=0.5,
+        )
+    except TimeoutError:
+        console.log("[red]Failed when trying to retrieve poll message")
+        return
+    # Get poll number of reactions in count order
+    poll_votes = []
+    for emoji, users in message.get_reactions().items():
+        # Remove bot reactions from count
+        users.remove(bot_id)
+        poll_votes.append([emoji, len(users)])
+    poll_votes = sorted(poll_votes, reverse=True, key=lambda x: x[1])
+
+    most_voted = poll_votes[0][1]
+
+    # Reply on chat to show the results
+    # Send poll text as reply to original message
+    msg_body = ">{0}".format(message.get_body()).replace("\n", "\n>")
+    msg_body += "\n## Poll results:"
```

Comment:
The only thing I'm not sure I love is that the bot replies to the original poll message as it takes up a lot of space. Maybe we either take that out or make it a boolean setting in `brainbot.ini` so that it's optional.

Reply:
I can remove it/make it optional with no problems but I honestly think it should stay, when making a quick poll everyone knows what poll the results correspons to, but when scheduling polls for longer periods of time can be confusing to receive an empty poll results without any context whatsoever.

Let me know if you still want me to modify it :)

Reply:
Ah, I see what you're saying. What about parsing out the title and options from the original message to include in the results? Do you think that would make things too complex?

Reply:
That's much better! Actually, if I remember well I made it so the title and options are saved and parsed in the reminder task.
I'll see if I can add the modifications between today and tomorrow.